src/components/Kv/KvPillToggle.vue

```diff
@@ -23,10 +23,15 @@ export default {
 			type: String,
 			required: true
 		},
+		callback: {
+			type: Function,
+			default: null,
+		}
 	},
 	methods: {
-		pillToggled(key) {
-			this.$emit('pill-toggled', key);
+		pillToggled(key, callback) {
+			// eslint-disable-next-line quote-props
+			this.$emit('pill-toggled', { 'key': key, 'callback': callback });
 		}
```

Comment:
We’ll have to find a different solution for this. Passing a function to a component then getting it back from that component isn’t a pattern that will be reused and as such doesn’t belong in a reusable widget.

Reply:
My suggestion is passing the refine function as a prop to a component that encapsulates the toggle. Then when the toggle sends the event we can call this.refine.

Reply:
@ksekhri PR updated with your suggestions. Thanks!